### PR TITLE
Required fixes

### PR DESCRIPTION
**Pull request by Infinical**

### CODE REVIEW

File: Amigolden/src/app/components/map/map.component.ts [hunk 1]
```diff
@@ -31,54 +41,66 @@ export class MapComponent implements OnInit {
   withinMiles = 10;
   latitude = 40.7362942;
   longitude = -73.9921495;
+
   zoom: number;
-  address: string;
+  address: string = '';
```
Comment (Amigolden) [inline Amigolden/src/app/components/map/map.component.ts]: remove type, the string is inferred from the assignment

File: Amigolden/src/app/components/map/map.component.ts [hunk 2]
```diff
@@ -31,54 +41,66 @@ export class MapComponent implements OnInit {
...
+  town: string = '';
```
Comment (Amigolden) [inline Amigolden/src/app/components/map/map.component.ts]: remove type

File: Amigolden/src/app/components/map/map.component.ts [hunk 3]
```diff
@@ -31,54 +41,66 @@ export class MapComponent implements OnInit {
...
+  state: string = '';
```
Comment (Amigolden) [inline Amigolden/src/app/components/map/map.component.ts]: remove type

File: Amigolden/src/app/components/shared.components.module.ts [hunk 4]
```diff
@@ -1,30 +1,46 @@
...
+import { FormsModule, ReactiveFormsModule } from "@angular/forms";
+import { IonicModule } from "@ionic/angular";
+import { InlineEditComponent } from "./inline-edit/inline-edit";
+import { StripeCardComponent } from "./stripe-card/stripe-card.component";
+import { AgmCoreModule } from "@agm/core";
+import { MapComponent } from "./map/map.component";
+import { AgmSnazzyInfoWindowModule } from "@agm/snazzy-info-window";
+import { VoteListPage } from "../pages/vote/vote-list/vote-list.page";
+import { ProfilePictureUploadComponent } from "./profile-picture-upload/profile-picture-upload";
+import { ImageCropperModule } from "ngx-image-cropper";
```
Comment (Amigolden) [inline Amigolden/src/app/components/shared.components.module.ts]: remove this file from the commit

File: Amigolden/src/app/pages/messages/messages.page.ts [hunk 5]
```diff
@@ -1,128 +1,148 @@
...
-      this.baseProvider.getDynamicList(this.filterByConversationId,
-        pagingInfo.pageSize, pagingInfo.pageNumber).pipe(map(message => {
+  public config = new ListConfiguration<Message>(
+    (pagingInfo: PagingInfo) =>
+      this.baseProvider
+        .getDynamicList(
+          this.filterByConversationId,
+          pagingInfo.pageSize,
+          pagingInfo.pageNumber
+        )
```
Comment (Amigolden) [inline Amigolden/src/app/pages/messages/messages.page.ts]: this file shouldn't have changes remove it from the commit

**Comment by Amigolden** [comment]
The map should take up the full screen while still leaving enough room for the footer

**Review by Amigolden — CHANGES_REQUESTED** [review]

File: Amigolden/src/app/pages/messages/messages.page.ts (hunk 5 above)
Comment (Infinical) [reply Amigolden/src/app/pages/messages/messages.page.ts]: removed

File: Amigolden/src/app/components/shared.components.module.ts (hunk 4 above)
Comment (Infinical) [reply Amigolden/src/app/components/shared.components.module.ts]: removed

File: Amigolden/src/app/components/map/map.component.scss [hunk 6]
```diff
@@ -4,14 +4,14 @@
...
-  min-height: 80vh;
+  height: 70vh;
+  min-height: 70vh;
 }
 
 
 #footer {
-  height: 10vh;
-  min-height: 10vh;
+  height: 20vh;
```
Comment (Amigolden) [inline Amigolden/src/app/components/map/map.component.scss]: the footer should be a static pixel size, the food only contain the address it doesnt need to flex, set this to be some pixel size 100px or whatever looks nice there

File: Amigolden/src/app/components/map/map.component.scss [hunk 7]
```diff
@@ -1,3 +1,45 @@
...
-    height: 300px;
-}
\ No newline at end of file
+#map-container {
+  height: 90vh;
+  min-height: 90vh;
+  
+}
+ #map {
+  height: 70vh;
```
Comment (Amigolden) [inline Amigolden/src/app/components/map/map.component.scss]: the map should not be proportional to the ViewHeight, the map size should be proportional to its parent container.

File: Amigolden/src/app/components/map/map.component.scss [hunk 8]
```diff
@@ -1,3 +1,45 @@
...
+
+.image {
+  display: block;
+  width: 100%;
+  height: auto;
+}
+
+.overlay{
+  position: absolute;
+  top: 3vh;
```
Comment (Amigolden) [inline Amigolden/src/app/components/map/map.component.scss]: Vh is not the proper measurements.

File: Amigolden/src/app/components/map/map.component.ts [hunk 9]
```diff
@@ -31,54 +43,199 @@ export class MapComponent implements OnInit {
...
-  public searchElementRef: ElementRef;
+  slidePrev() {
+    this.slides.slidePrev();
+  }
+  slideNext() {
+    this.slides.slideNext();
+  }
+
+  //Flip
+  slideOptions = {
```
Comment (Amigolden) [inline Amigolden/src/app/components/map/map.component.ts:60]: please conform to the tslint.json, you should be able to see the error in the project if you're using VSCode. Declaration of instance field is not allowed after methods

File: Amigolden/src/app/components/map/map.component.ts [hunk 10]
```diff
@@ -31,54 +43,199 @@ export class MapComponent implements OnInit {
...
+          .transition(duration)
+          .find(
+            '.swiper-slide-shadow-top, .swiper-slide-shadow-right, .swiper-slide-shadow-bottom, .swiper-slide-shadow-left'
+          )
+          .transition(duration);
+        if (swiper.params.virtualTranslate && duration !== 0) {
+          let eventTriggered = false;
+          // eslint-disable-next-line
+          slides.eq(activeIndex).transitionEnd(function onTransitionEnd() {
+            if (eventTriggered) return;
```
Comment (Amigolden) [inline Amigolden/src/app/components/map/map.component.ts]: if statements should be wrapped in {}, please conform to ts lint standards

File: Amigolden/src/app/components/shared.components.module.ts (hunk 4 above)
Comment (Amigolden) [reply Amigolden/src/app/components/shared.components.module.ts]: Its still here

**Review by Amigolden — CHANGES_REQUESTED** [review]

**Pull request closed without merging**